scripts/index_docs.py

```
@@ -0,0 +1,196 @@
+        print(f"  -> {len(chunks)} chunks")
+        all_chunks.extend(chunks)
+
+    print(f"\nTotal chunks to embed: {len(all_chunks)}")
+
+    # Embed each chunk
+    print(f"Embedding with model: {embedding_model}")
+    for i, chunk in enumerate(all_chunks):
+        embed_input = f"[{chunk['source']} / {chunk['heading']}]\n{chunk['content']}"
+        try:
+            chunk["embedding"] = embed_text(embed_input, embedding_model, api_key)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)